### PR TITLE
Huge 0.3.0 release; More tests, more stable, more fixes, added auto replace fetching! 😱

### PR DESCRIPTION
**Pull request by bwplotka**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,9 +1,12 @@
 module github.com/bwplotka/bingo
 
-go 1.14
+go 1.15
```
Comment (kakkoyun) [inline go.mod:3]: Do you actually use any features specific to go 1.15?

**Review by kakkoyun — COMMENTED** [review]
Looking good. It's a huge one, it was really even hard to skim through it.

**Comment by bwplotka** [comment]
Thanks, don't expect you to look at all things. Thanks so much for review 🤗 @kakkoyun

File: go.mod (hunk 1 above)
Comment (bwplotka) [reply go.mod:3]: Not really good point

**Comment by bwplotka** [comment]
Lint will fail - mdox is flaky at the mo (:

**Pull request merged**